frontend/src/views/AboutView.vue

```diff
@@ -1,3 +1,214 @@
+              <v-card class="pa-2 pr-2 mx-auto" tile>
+                <v-col align="center" class="mt-5">
+                  <v-avatar size="100">
+                    <v-img class="photos" v-bind:src="tvideiraPic"></v-img>
+                  </v-avatar>
+                  <v-card-text class="text--primary">
+                    tvideira<br />name?<br />
+                    <div>
+                      <a
+                        v-bind:href="'https://github.com/tvideira'"
+                        target="_blank"
+                      >
+                        <img
+                          id="github"
+                          style="max-height: 30px"
+                          v-bind:src="githubLogo"
+                          v-bind:alt="githubAlt"
+                        />
+                      </a>
+                    </div>
+                    [other, like mail?]
+                  </v-card-text>
+                </v-col>
+              </v-card>
+            </v-col>
+            <v-col class="pl-2 pr-2 pt-4">
+              <v-card class="pa-2 pr-2 mx-auto" tile>
+                <v-col align="center" class="mt-5">
+                  <v-avatar size="100">
+                    <v-img class="photos" v-bind:src="abouchauraPic"></v-img>
+                  </v-avatar>
+                  <v-card-text class="text--primary">
+                    abouchau<br />name?<br />
+                    <div>
+                      <a
+                        v-bind:href="'https://github.com/N-4-Nina'"
+                        target="_blank"
+                      >
+                        <img
+                          id="github"
+                          style="max-height: 30px"
+                          v-bind:src="githubLogo"
+                          v-bind:alt="githubAlt"
+                        />
+                      </a>
+                    </div>
+                    [other, like mail?]
+                  </v-card-text>
+                </v-col>
+              </v-card>
+            </v-col>
+            <v-col class="pl-2 pr-2 pt-4">
+              <v-card class="pa-2 pr-2 mx-auto" tile>
+                <v-col align="center" class="mt-5">
+                  <v-avatar size="100">
+                    <v-img class="photos" v-bind:src="tmorrisPic"></v-img>
+                  </v-avatar>
+                  <v-card-text class="text--primary">
+                    tmorris<br />name?<br />
+                    <div>
+                      <a
+                        v-bind:href="'https://github.com/tmorris42'"
+                        target="_blank"
+                      >
+                        <img
+                          id="github"
+                          style="max-height: 30px"
+                          v-bind:src="githubLogo"
+                          v-bind:alt="githubAlt"
+                        />
+                      </a>
+                    </div>
+                    [other, like mail?]
```

Comment:
For me, I'd leave it without any other links, but again it's a good idea, and we can check with the team